packages/apptive_grid_form/lib/src/widgets/form_widget/form_content_widgets.dart

```diff
@@ -139,14 +140,20 @@ class FormDataWidget extends StatelessWidget {
           } else if (index < (data.components?.length ?? 0) + indexOffset) {
             final componentIndex = index - indexOffset;
             final component = data.components![componentIndex];
+            final properties = data.fieldProperties
+                .firstWhereOrNull((e) => e.fieldId == component.field.id);
+            component.data.value ??= properties?.defaultValue?.value;
+            if (properties?.hidden == true) {
+              return const SizedBox();
+            }
             final componentWidget = fromModel(component);
             if (componentWidget is EmptyFormWidget) {
               // UserReference Widget should be invisible in the Form
               // So returning without any Padding
               return componentWidget;
             } else {
               return IgnorePointer(
-                ignoring: isSubmitting,
+                ignoring: properties?.disabled == true || isSubmitting,
```

Comment:
Not 100% so please check if `IgnorePointer` also makes it skipped when using focus switching eg with `Next` on a Softkeyboard and/or `tab` on a hardware keyboard

Reply:
It does work (tab doesn't jump to it), but while testing it I realised, we need to change the UI to show the user, that the widget is disabled